app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -28,9 +28,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("VisualVerificationActivity");
     private static final int SECURITY_CODE_LENGTH = 5;
+    private static final int NUM_BUTTONS = 12;
```

Comment:
Kinda scary to have it hardcoded as our project is open source...
Then again no one will probably use this version of the app 🤔